### PR TITLE
[11.0][ADD] pos_cashier_login

### PR DESCRIPTION
**Pull request by kirca**
Require cashier to be scanned before each new PoS order.

pos_cashier_login

### CODE REVIEW

File: pos_cashier_login/static/src/js/main.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+odoo.define('pos_cashier_login.main', function (require) {
+    'use strict';
+
+    var gui = require('point_of_sale.gui');
+    var screens = require('point_of_sale.screens');
+    var chrome = require('point_of_sale.chrome');
+    var pos_models = require('point_of_sale.models');
+    var SuperPosModel = pos_models.PosModel.prototype;
+
+    pos_models.PosModel = pos_models.PosModel.extend({
```
Comment (hparfr) [inline pos_cashier_login/static/src/js/main.js:12]: you may use an include instead:
```js
pos_models.PosModel.include({
...
})
```

**Review by hparfr — APPROVED** [review]
The code looks good to me.

File: pos_cashier_login/static/src/js/main.js (hunk 1 above)
Comment (kirca) [reply pos_cashier_login/static/src/js/main.js:12]: Tried but it is a backbone model and it doesn't have such a method.

**Comment by kirca** [comment]
@hparfr thanks for review!

File: pos_cashier_login/static/src/js/main.js (hunk 1 above)
Comment (hparfr) [reply pos_cashier_login/static/src/js/main.js:12]: ok :)

**Review by PierrickBrun — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Thanks @kirca :+1:

**Pull request merged**